doc/classes/String.xml

```diff
@@ -831,33 +914,33 @@
 		<method name="to_utf16_buffer" qualifiers="const">
 			<return type="PackedByteArray" />
 			<description>
-				Converts the String (which is an array of characters) to UTF-16 encoded [PackedByteArray] (which is an array of bytes).
+				Converts the string to an [url=https://en.wikipedia.org/wiki/UTF-16]UTF-16[/url] encoded [PackedByteArray].
```

Comment:
Although U in UTF is a vowel, it's spoken as "you-tee-eff" which starts a consonant, requiring an "a" instead of "an". This is similar to how you say "an MP3 file".

Same with the other UTF's